lib/dojos.js

```diff
@@ -323,6 +323,7 @@ exports.register = function (server, options, next) {
             alpha3: joiValidator.alpha3().required(),
             address1: Joi.string().required(),
             coordinates: Joi.string().required(),
+            attendee: Joi.string().required(),
```

Comment:
I would have preferred something as expectedAttendees to avoid misunderstanding upon the goal of this attribute.
When i read the code, i thought it was required to have an attendee as a string when creating a dojo, sounded strange :p
Also, shoudn't validation be a number instead of a string?